Dockerfile.armhf

```diff
@@ -0,0 +1,13 @@
+FROM armhf/alpine:latest
+
+ENV GOPATH /go
+
+RUN apk add --update --no-cache curl ca-certificates git golang && \
```

Comment:
The package name should be `go`

https://pkgs.alpinelinux.org/flag/v3.4/community/go/1.6.3-r0